datacenter/ucp/2.1/guides/admin/configure/add-sans-to-cluster.md

```diff
@@ -0,0 +1,45 @@
+---
+title: Add SANs to cluster certificates
+description: Learn how to add new SANs to cluster nodes, allowing you to connect to UCP with a different hostname
+keywords: Docker, cluster, nodes, labels, certificates, SANs
+---
+
+When you connect to UCP, you need to make sure that the hostname that you use to connect is recognized
```

Comment:
This is looking great, but can we structure this intro to make it a bit more obvious there there are two choices. I would structure the intro like this

1. UCP has HTTPS out of the box. This means that you need to make sure the TLS certificates have information about the hostname you're using to access UCP. (You already cover this)
2. If you're using your own TLS certs, make sure they respect this, and learn more at `/datacenter/ucp/2.1/guides/admin/configure/use-your-own-tls-certificates.md`
3. If you want to use the self-signed certificate that UCP has out of the box, ensure the certificate respects this. Here's how you do it. (You already cover this)

If you don't do a good job of explaining these two "modes" most users will strugle with it and continue asking support for help.
Now that I think about it, I can configure UCP to use my own TLS certs, but then go to the node and add more SANS. Can you check if this still happens and file an issue with UCP so that we can make this more obvious?